eq-author/src/App/history/HistoryPage.js

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import styled from "styled-components";
+import { useQuery } from "@apollo/react-hooks";
+
+import CustomPropTypes from "custom-prop-types";
+
+import Error from "components/Error";
+import Loading from "components/Loading";
+import ScrollPane from "components/ScrollPane";
+import MainCanvas from "components/MainCanvas";
+import HistoryItem from "./HistoryItem";
+import Header from "components/EditorLayout/Header";
+import { Grid } from "components/Grid";
+import questionnaireHistoryQuery from "./questionnaireHistory.graphql";
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+`;
+
+const StyledGrid = styled(Grid)`
+  display: flex;
+  flex-direction: column;
+  padding: 1em 1em 2em 1em;
+  border: 1px solid #d6d8da;
+  margin: 0.5em 1em;
+`;
+
+export const HistoryPageContent = ({ match }) => {
+  const { loading, error, data } = useQuery(questionnaireHistoryQuery, {
+    variables: { input: { questionnaireId: match.params.questionnaireId } },
+  });
+
+  if (loading) {
+    return <Loading height="100%">Questionnaire history loading…</Loading>;
+  }
+
+  if (error) {
+    return <Error>Oops! Something went wrong</Error>;
```

Comment:
I think the error text needs to be a little more descriptive than this; at the moment it doesn't give any indication as to what went wrong or where. This could make solving potential issues down the line a bit harder.

Could we include any information from the `error` var in the message?

Reply:
I agree that it should be a more descriptive message. But this is message we are displaying when an error occurs in the app (at the moment) :/